ci/test_cpp.sh

```diff
@@ -37,5 +37,21 @@ if (( ${SUITEERROR} == 0 )); then
     SUITEERROR=$?
 fi
 
+if (( ${SUITEERROR} == 0 )); then
+    # Run a small Google benchmark to test that benchmarks are runnable
+    pushd $CONDA_PREFIX/bin/benchmarks/libcudf
+    ./MERGE_BENCH --benchmark_filter=/2/
+    SUITEERROR=$?
+    popd
+fi
+
+if (( ${SUITEERROR} == 0 )); then
+    # Run a small nvbench benchmark to test that benchmarks are runnable
+    pushd $CONDA_PREFIX/bin/benchmarks/libcudf
+    ./STRINGS_NVBENCH --run-once --benchmark 0
```

Comment:
Does anyone know how nvbench / Google benchmark handle exit codes? It's my hope that they always return `0` for success if the benchmark completes. These are very minimal benchmarks just to test that the executables are valid. I don't want bad performance or noise on the CI system to cause random failures.

Reply:
Yes the benchmarks will always return `0` for success